packages/gatsby/src/redux/types.ts

```diff
@@ -101,9 +101,11 @@ export interface IGatsbyJobContent {
 }
 
 export interface IGatsbyJobV2 {
-  job: IGatsbyJobContent
-  plugin: IGatsbyPlugin
+  job?: IGatsbyJobContent
+  plugin?: IGatsbyPlugin
   traceId?: string
+  result?: string[]
+  inputPaths?: string[]
 }
```

Comment:
What's the reason for these changes?

Reply:
I think that were some types that I didn't get right at the beggining, but now things are becoming more clear to me.

I'm removing the optional for the `job`, `plugin`, `result` and `inputPaths` keys inside this interface now. Since we actually set it inside the reducer we actually need the `result` and `inputPaths` in this interface.  Am I right?